src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -1251,6 +1251,7 @@ public void Evict<T>(T entity)
                 DocumentsByEntity.Evict(entity);
                 DocumentsById.Remove(documentInfo.Id);
                 _countersByDocId?.Remove(documentInfo.Id);
+                _timeSeriesByDocId?.Remove(documentInfo.Id);
```

Comment:
Do we need to clear the cache as well?

Reply:
what do you mean? this is the time series cache

Reply:
Right, somehow I thought there a different entity for TS caching.